commodore/catalog.py

```diff
@@ -175,6 +175,24 @@ def _kapitan_029_030_difffunc(
     before_text: str, after_text: str, fromfile: str = "", tofile: str = ""
 ) -> tuple[Iterable[str], bool]:
 
+    [diff, _] = _ignore_non_semantic_whitespace_difffunc(
```

Comment:
No need to create a useless list to unpack the tuple return value, just using the following should work fine:

```suggestion
    diff, _ = _ignore_non_semantic_whitespace_difffunc(
```

Reply:
That would be my ruby background 🙈